src/Execution/Executor.php

```diff
@@ -946,6 +957,15 @@ protected function createResolveInfo(
         );
     }
 
+    /**
+     * @param ExecutionException $error
+     */
+    protected function handleError(ExecutionException $error)
```

Comment:
`: void`

Reply:
We don't use that anywhere yet, we should though, but I'll keep it like this for consistency.

Reply:
It's used in the same class

Reply:
Where? I can't see it anywhere.